rulesets/google-event-measurement.js

```diff
@@ -1,65 +1,12 @@
-window['_dlo_rules_google_measurement'] = [
+window["_dlo_rules_google_em"] = [
```

Comment:
I updated this ruleset to match the ruleset being served with FullStory data layer capture integrations.